hack/pin-update.sh

```
@@ -0,0 +1,50 @@
+# Defaults
+MTV_SUBSCRIPTION=mtv-operator
+MTV_NAMESPACE=openshift-mtv
+MTV_VERSION=2.9.2 # change to the version you want to upgrade to
+CATALOG_SOURCE=redhat-operators
+
+# Get installed CSV from current subscription
+CSV=$(oc get subscription $MTV_SUBSCRIPTION -n $MTV_NAMESPACE -o json | jq -r '.status.installedCSV')
+# Remove the current subscription
+oc delete subscription -n $MTV_NAMESPACE $MTV_SUBSCRIPTION
+# Remove the current CSV
+oc delete csv -n $MTV_NAMESPACE $CSV
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)